pkg/cryptoauthlib/doc.txt

```diff
@@ -0,0 +1,58 @@
+/**
+ * @defgroup pkg_cryptoauthlib cryptoauthlib security crypto 
```

Comment:
not sure if defining such generic groups (i.e. security and crypto) here is a good idea. 

Also defining multiple groups in one line does not work anyway - see CI errors, too.